packages/zoltan2/test/connectivity/meshData/test1-borders.quad.msh

```diff
@@ -0,0 +1,10 @@
+nonsense line
+0 0 8
```

Comment:
This file does not follow the file format, which prescribes that line 3 must be blank.  Does it actually run correctly?  If so, why?

Reply:
The format has changed, so this is no longer an issue.